README.md

```diff
@@ -141,6 +141,8 @@ An object used to automatically send your build data to Datadog.
 
 ![](./assets/datadog-dashboard.png)
 
+> You can get this dashboard's configuration by running `yarn cli dashboard --prefix <your.prefix>`.
+
 The most basic configuration looks like this, consult
```

Comment:
We might want to provide a default prefix inside the basic configuration below if the default of `""` doesn't create the intended results.

Reply:
That will be done, if needed, in a different PR.